.ebextensions/02_data_downloads.config

```diff
@@ -1,46 +1,22 @@
 commands:
-    01_install_postgresql_devel:
```

Comment:
I don't think `01_install_postgresql_devel` is needed anymore since it's in `01_build.config`?

Reply:
hopefully!

Reply:
We'll find out 😂